### PR TITLE
Fix meta tags

### PR DESCRIPTION
**Pull request by G-Ambatte**
This PR fixes the unexpected behaviour of the default meta tag values. It also changes the default type to article from website, based on initial feedback.

### CODE REVIEW

File: client/template.js [hunk 1]
```diff
@@ -11,10 +11,10 @@ module.exports = async(name, title = '', props = {})=>{
 		<link rel="icon" href="/assets/homebrew/favicon.ico" type="image/x-icon" />
 		<meta property="og:title" content="${props.brew?.title ?? 'Homebrewery - Untitled Brew'}">
 		<meta property="og:url" content="${HOMEBREWERY_PUBLIC_URL}/${props.brew?.shareId ? `share/${props.brew.shareId}` : ''}">
-		<meta property="og:image" content="${props.brew?.thumbnail ?? `${HOMEBREWERY_PUBLIC_URL}/thumbnail.png`}">
-		<meta property="og:description" content="${props.brew?.description ?? 'No description.'}">
+		<meta property="og:image" content="${props.brew?.thumbnail ? props.brew?.thumbnail : `${HOMEBREWERY_PUBLIC_URL}/thumbnail.png`}">
```
Comment (calculuschild) [inline client/template.js:14]: Why the change here? The logic should be identical.

Comment (G-Ambatte) [reply client/template.js]: It looks like switching to `||` fixed the issue as well. I've just pushed a PR using that operator instead.
